playbooks/roles/mongo_newrelic_monitor/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+MONGO_NEWRELIC_MONITOR_USER: 'newrelic-monitor'
+MONGO_NEWRELIC_MONITOR_PASSWORD: 'CHANGE_ME'
```

Comment:
@swalladge A more widely used placeholder for secure credentials is [`SET-ME-PLEASE`](https://github.com/edx/configuration/search?q=SET-ME-PLEASE&unscoped_q=SET-ME-PLEASE). Let's use that here as well :)